tests/resources/test_project.py

```diff
@@ -339,6 +578,27 @@ def test_list_projects(collection, session):
     assert 5 == len(projects)
 
 
+def test_list_projects_v3(collection_v3, session_v3):
+    # Given
+    projects_data = ProjectDataFactory.create_batch(5)
+    session_v3.set_response({'projects': projects_data})
+
+    # When
+    projects = list(collection_v3.list())
```

Comment:
Is the `team_id` argument supposed to be required in the v3 API?

Reply:
the team_id is set in the creation of the collection, so there's no need for the user to specify team_id in the call (the docs might be out of date, this was a change in the design mid pr)

Reply:
Gotcha, gotcha. Can you just update this Collection methods requirement in the ticket, then?

`citrine.projects.list           -  add an optional team_id kwarg that is required for v3`

Reply:
updated

Reply:
I see the update; thank you.